pages/database-management/logs.mdx

```diff
@@ -114,6 +114,23 @@ container:
 
 </Steps>
 
+## Hiding sensitive information (Enterprise)

```

Comment:
Are there any other kinds of info that are hidden? Or is it only passwords?

Reply:
only passwords

Reply:
Then say "Hiding passwords" instead of "hiding sensitive information"